octobot/backtesting/abstract_backtesting_test.pxd

```diff
@@ -26,5 +26,4 @@ cdef class AbstractBacktestingTest:
                                         object tentacles_setup_config,
                                         dict config)
 
-    cdef void _handle_results(self, object independent_backtesting, object profitability)
```

Comment:
The "real" fix would have been to return something instead of void. But it also works like this.

Reply:
Not sure how it would fix it with void, the issue is that local implementations (strategy tests) are not cythonized (which is normal) therefore the cythonized version of the method is used and it's what is raising this error (this error should not be raised)

Reply:
Cython is silencing exception when a cythonized method doesn't return anything. It's not related to how the method is called.

Reply:
Yes but the issue is that the wrong method is called, it should not be called at all and the cython declaration is preventing this call

Reply:
Oh ok, understood